test/suites/auth.sh

```diff
@@ -1145,6 +1152,65 @@ auth_project_features() {
   lxc project delete blah
 }
 
+auth_ovn() {
+  remote="${1}"
+
+  if [ -z "${LXD_OVN_NB_CONNECTION:-}" ]; then
+    echo "OVN northbound connection not set. Skipping OVN tests..."
+    return
+  fi
+
+  lxc config set network.ovn.northbound_connection "${LXD_OVN_NB_CONNECTION}"
+
+  # If the connection uses SSL we have more required environment variables.
+  # Set the client cert, key, and CA cert.
+  if [[ "${LXD_OVN_NB_CONNECTION}" =~ ^ssl: ]]; then
+    lxc config set network.ovn.client_cert="$(< "${LXD_OVN_NB_CLIENT_CRT_FILE}")"
+    lxc config set network.ovn.client_key="$(< "${LXD_OVN_NB_CLIENT_KEY_FILE}")"
+    lxc config set network.ovn.ca_cert="$(< "${LXD_OVN_NB_CA_CRT_FILE}")"
```

Comment:
I wonder if there would be a benefit in doing those config changes in one go.
```suggestion
    lxc config set network.ovn.client_cert="$(< "${LXD_OVN_NB_CLIENT_CRT_FILE}")" network.ovn.client_key="$(< "${LXD_OVN_NB_CLIENT_KEY_FILE}")" network.ovn.ca_cert="$(< "${LXD_OVN_NB_CA_CRT_FILE}")"
```
Would it avoid bounding OVN multiple times or something like that?